console-subscriber/src/builder.rs

```diff
@@ -145,6 +149,16 @@ impl Builder {
         }
     }
 
+    /// Sets the filter environment variable for tracing events.
+    ///
+    /// By default, this is `RUST_LOG`.
```

Comment:
some notes on the documentation here:
 - this specifically filters the tracing events that are _logged to stdout_ by the `fmt` layer added in `Builder::init`, it does not effect the events that are recorded by the console itself. I think the documentation should probably state that explicitly. 
- only `Builder::init` adds a logging layer; `Builder::build` and `Builder::spawn` will not. I think this documentation should state that clearly.
- there should probably be a link to the `tracing` documentation explaining the syntax for the environment variable.

here's my suggestion:
```suggestion
    /// Sets the environment variable used to configure which `tracing` events
    /// are logged to stdout.
    ///
    /// The [`Builder::init`] method configures the default `tracing`
    /// subscriber. In addition to a [`ConsoleLayer`], the subscriber
    /// constructed by `init` includes a [`fmt::Layer`] for logging events to
    /// stdout. What `tracing` events that layer will log is determined by the
    /// value of an environment variable; this method configures which
    /// environment variable is read to determine the log filter.
    ///
    /// This environment variable does not effect what spans and events are
    /// recorded by the [`ConsoleLayer`]. Therefore, this method will have no
    /// effect if the builder is used with [`Builder::spawn`] or
    /// [`Builder::build`].
    ///
    /// The default environment variable is `RUST_LOG`. See [here] for details
    /// on the syntax for configuring the filter.
    ///
    /// [`fmt::Layer`]: https://docs.rs/tracing-subscriber/0.3/tracing_subscriber/fmt/index.html
    /// [here]: https://docs.rs/tracing-subscriber/0.3/tracing_subscriber/filter/targets/struct.Targets.html
```